build.sh

```diff
@@ -3,51 +3,8 @@
 # Copyright (c) .NET Foundation and contributors. All rights reserved.
 # Licensed under the MIT license. See LICENSE file in the project root for full license information.
 #
-
-# Set OFFLINE environment variable to build offline
-
 set -e
```

Comment:
OFFLINE is not used anywhere else in the repository, so the comment is misleading :-(.